hosts/Makefile

```diff
@@ -9,7 +9,8 @@ substrate:
 	cp substrate/target/release/polkadot ../bin/
 
 kagome:
-	cmake -DCMAKE_BUILD_TYPE=Release -S kagome -B kagome/build
+  cmake -DCMAKE_BUILD_TYPE=Release -DHUNTER_CONFIGURATION_TYPES=Release \
+-S kagome -B kagome/build
```

Comment:
The line break here does not seem supported by GNU Make and breaks the CI. Please put everything on one line.